definitions/infra-awscloudfrontdistribution/golden_metrics.yml

```diff
@@ -14,3 +14,19 @@ totalErrorRate:
     where: provider='CloudFrontDistribution'
     facet: entityName
     eventId: entityGuid
+bytesUploaded:
```

Comment:
this map key is duplicated and it causes the checks to fail.

Reply:
Sorry, thanks for noticing @IreneMLC